sageresearch-app-sdk/src/main/java/org/sagebionetworks/research/sageresearch/dao/room/ResourceEntity.kt

```diff
@@ -0,0 +1,51 @@
+package org.sagebionetworks.research.sageresearch.dao.room
+
+import android.arch.persistence.room.ColumnInfo
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.PrimaryKey
+import com.google.gson.annotations.SerializedName
+import org.sagebionetworks.bridge.rest.RestUtils
+import java.lang.reflect.Type
+
+@Entity
+data class ResourceEntity(
+
+    /**
+     * @property identifier the resource identifier
+     */
+    @PrimaryKey
+    var identifier: String,
```

Comment:
What do you think about making a composite primary key of identifier + type? In some cases our Bridge identifiers aren't GUIDs